.github/workflows/build.yml

```
@@ -249,10 +249,45 @@ jobs:
           set PATH=%PATH%;%GITHUB_WORKSPACE%\ACE_TAO\ACE\lib;%GITHUB_WORKSPACE%\OpenDDS\lib;%GITHUB_WORKSPACE%\tools\v8stubs"
           echo "PATH = %PATH%"
           msbuild -p:Configuration=Debug,Platform=x64 -m node_opendds.sln
+      - name: 'Test (windows)'
+        if: ${{ matrix.m.os == 'windows-2019' }}
+        shell: cmd
+        run: |-
+          cd OpenDDS
+          call setenv.cmd
+          cd ..
+          set /p V8_ROOT=<v8_path.txt
+          set V8_ROOT=%V8_ROOT%\..\..
+          echo "V8_ROOT = %V8_ROOT%"
```

```suggestion
          echo V8_ROOT = %V8_ROOT%
```
Or "set V8" shows all environment variables starting with V8.